### DISCUSSION TITLE
Error Fetching /opt/homebrew/Library/Taps/heroku/homebrew-brew failed!

### DISCUSSION THREAD

**Discussion by Pres2000** [Homebrew/discussions#5316] [Everyday usage]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

Hey anyone run into this issue before or know what is going on here?
ran the command: /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
to install Brew, but keep getting this error.

**Comment by carlocab** [comment]
Are you on Linux or an M1 Mac? If so, did you follow the instructions about adding `brew` to your `PATH`? This was shown at the end of the install script.
